sync/class.jetpack-sync-functions.php

```diff
@@ -293,7 +293,7 @@ public static function get_plugins_action_links( $plugin_file_singular = null )
 			if ( is_null( $plugin_file_singular ) ) {
 				return $plugins_action_links;
 			}
-			return ( isset( $plugins_action_links[ $plugin_file_singular ] ) ? $plugins_action_links[ $plugin_file_singular ] : array() );
+			return ( isset( $plugins_action_links[ $plugin_file_singular ] ) ? $plugins_action_links[ $plugin_file_singular ] : null );
 		}
 		return array();
```

Comment:
should this be null?

Reply:
No since that is what gets returned by the function. This way we don't always have to check for is_array and can iterate over it easier without errors.